clipboard.py

```diff
@@ -1,47 +1,148 @@
 import torch
```

Comment:
Please add the purpose of this as docstring

Reply:
Sorry. When I was testing with new functions in clipboard.py , the comment describing the information of this docstring got erased. `clipboard.py` is mainly used as a placeholder for testing out functions that I don't want to incorporate into the main code base yet. After testing the new functions that I want to incorporate, then I swap out the functions in main codebase with new functions from clipboard.